Model/Request.php

```diff
@@ -98,5 +98,10 @@ private function validateResponse(ClientInterface $client): void
         if ($code && $message) {
             throw new ResponseException(__($message));
         }
+
+        if (!$code && $message) {
+            throw new ResponseException(__('Server Error'));
+        }
```

Comment:
Can we shorten this to one if condition?


```suggestion
        if ($message) {
            throw new ResponseException(__($message));
        }
```

Reply:
@DiskoPete I guess so. I was just adding things in a way to be sure not to break anything ;)